about.css

```diff
@@ -0,0 +1,319 @@
+}
+
+.logo-icon {
+  display: flex;
+}
+
+.lists a {
+  text-decoration: none;
+  color: #282828;
+  list-style: none;
+  margin-right: 50px;
+}
+
+.about-link a {
+  text-decoration: none;
+  color: #f55114;
+}
+
+.home a {
+  display: flex;
+  justify-content: right;
+  text-decoration: none;
+  color: #f55114;
+  padding: 40px;
+  gap: 3px;
+}
+
+.home i {
+  padding: 3px;
+}
+
+.home > a:hover {
+  color: #595959;
+}
+
+.lists > li > a:hover {
+  color: brown;
+}
+
+.lists {
+  list-style: none;
+  margin-top: 30px;
+}
+
+.logo {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  box-shadow: 0 0 10px #000;
+}
+
+.about-section {
+  background-image: url(./images/back.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.about-heading {
+  text-align: center;
+}
+
+.about-heading h1 {
+  color: #f55114;
+  font-weight: 700;
+  text-align: center;
+}
+
+.about-heading p {
+  background-color: #fff;
+  padding: 20px;
+  width: 60%;
+  margin: auto;
+  margin-top: 20px;
+}
+
+.about-link {
+  padding: 3rem;
+  text-decoration: none;
+  text-align: center;
+}
+
+.section2 {
+  background: #fff;
+  padding-top: 50px;
+  padding-bottom: 30px;
+}
+
+.about-text {
+  text-align: center;
+}
+
+#nft-logo {
+  justify-content: center;
+}
+
+.about-text2 {
+  text-align: center;
+  padding: 5rem;
+  background-color: #f4f4f4;
+}
+
+.card-container {
+  display: flex;
+  gap: 30px;
+  justify-content: center;
+}
+
+.card1 {
+  background-color: #f55114;
+  background-blend-mode: soft-light;
+  background-image: url(./images/conf.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 20rem;
+  height: 10rem;
+  color: #f4f4f4;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.card2 {
+  background-color: #f55114;
+  background-blend-mode: soft-light;
+  background-image: url(./images/conf2.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 20rem;
+  height: 10rem;
+  color: #f4f4f4;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.footer {
+  display: flex;
+  justify-content: space-around;
+  color: #f4f4f4;
+  padding: 10px;
+  background-color: #282828;
+}
+
+.footer-text {
+  margin-top: 15px;
+}
+
+.mobile-menu {
+  display: none;
+}
+
+.mobile-only {
+  display: none;
+}
+
+.nav-menu {
+  display: none;
+}
+
+/* the media query for the about page */
+@media screen and (max-width: 768px) {
+  .header {
+    display: none;
+  }
+
+  .home {
+    display: none;
+  }
+
+  /* the modal part */
+  .mobile-menu {
+    display: block;
+    padding-top: 50px;
+    padding-left: 50px;
+    font-size: xx-large;
+  }
+
+  .nav-menu {
+    display: block;
+    background-color: #3c3a39;
+    height: 800px;
+    width: 100%;
+    font-family: 'Inter', sans-serif;
+    position: absolute;
+    margin: 0;
+    padding: 0;
+    left: -200%;
+  }
+
+  .nav-menu.active {
+    left: 0;
+    transition: 0.5s;
+  }
```

Comment:
- [OPTIONAL] It's possible to scroll down when the nav menu is open the menu is not covering the entire screen in some screen sizes. Take a look on this issue.
<img width="502" alt="image" src="https://user-images.githubusercontent.com/61919617/181109513-6a1c2aa8-b4a9-47f9-9a42-589cf580e344.png">